src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -51,15 +55,17 @@ public class DatasetService {
     private final ConsentDAO consentDAO;
     private final DataAccessRequestDAO dataAccessRequestDAO;
     private final DatasetDAO datasetDAO;
+    private final GCSService gcsService;
     private final UserRoleDAO userRoleDAO;
     private final UseRestrictionConverter converter;
 
     @Inject
     public DatasetService(ConsentDAO consentDAO, DataAccessRequestDAO dataAccessRequestDAO, DatasetDAO dataSetDAO,
-                          UserRoleDAO userRoleDAO, UseRestrictionConverter converter) {
+                          GCSService gcsService, UserRoleDAO userRoleDAO, UseRestrictionConverter converter) {
         this.consentDAO = consentDAO;
         this.dataAccessRequestDAO = dataAccessRequestDAO;
         this.datasetDAO = dataSetDAO;
+        this.gcsService = gcsService;
```

Comment:
is gcsService needed? will it be needed in the follow up PR?

Reply:
It will be needed to store the upload file and yes, will be used in the next PR.